.github/ISSUE_TEMPLATE/contact-center-template.md

```diff
@@ -23,3 +27,6 @@ assignees: ''
 ## Support provided
 
 <!-- How was this issue ultimately solved? -->
+
+### List related GitHub tickets here
```

Comment:
I like this section - maybe a comment describing what GH tickets would go here? I'm imagining something like:

> Please list any existing tickets related to this issue. If ticket doesn't exist AND a long term solution is needed, please create that ticket and mention it here.